test/unit/Cover/coverNFT.js

```diff
@@ -9,98 +9,159 @@ describe('CoverNFT', function () {
-    const { members } = this.accounts;
-    await coverNFT.setMockOperator(members[0].address);
-    await coverNFT.connect(members[0]).mint(members[1].address, 0);
-    expect(await coverNFT.ownerOf(0)).to.be.equal(members[1].address);
+    const {
+      members: [operator, nftOwner],
+    } = this.accounts;
+    await coverNFT.setMockOperator(operator.address);
+    await coverNFT.connect(operator).mint(nftOwner.address, 0);
+    expect(await coverNFT.ownerOf(0)).to.be.equal(nftOwner.address);
   });
+
   it('should fail to burn - onlyOperator()', async function () {
     const { coverNFT } = this;
     await expectRevert(coverNFT.burn(0), 'CoverNFT: Not operator');
   });
+
   it('should successfully burn', async function () {
     const { coverNFT } = this;
-    const { members } = this.accounts;
-    await coverNFT.setMockOperator(members[0].address);
-    await coverNFT.connect(members[0]).mint(members[1].address, 0);
-    await coverNFT.connect(members[0]).burn(0);
+    const {
+      members: [operator, nftOwner],
+    } = this.accounts;
+    await coverNFT.setMockOperator(operator.address);
+    await coverNFT.connect(operator).mint(nftOwner.address, 0);
+    await coverNFT.connect(operator).burn(0);
     await expectRevert(coverNFT.ownerOf(0), 'NOT_MINTED');
   });
+
   it('should return success for isApproveOrOwner() - owner == sender', async function () {
     const { coverNFT } = this;
-    const { members } = this.accounts;
-    await coverNFT.setMockOperator(members[0].address);
-    await coverNFT.connect(members[0]).mint(members[1].address, 0);
-    expect(await coverNFT.isApprovedOrOwner(members[1].address, 0)).to.be.equal(true);
-    expect(await coverNFT.isApprovedOrOwner(members[0].address, 0)).to.be.equal(false);
+    const {
+      members: [operator, nftOwner],
+    } = this.accounts;
+    await coverNFT.setMockOperator(operator.address);
+    await coverNFT.connect(operator).mint(nftOwner.address, 0);
+    expect(await coverNFT.isApprovedOrOwner(nftOwner.address, 0)).to.be.equal(true);
+    expect(await coverNFT.isApprovedOrOwner(operator.address, 0)).to.be.equal(false);
   });
+
   it('should return success for isApproveOrOwner() - isApprovedForAll', async function () {
     const { coverNFT } = this;
-    const { members } = this.accounts;
-    await coverNFT.setMockOperator(members[0].address);
-    await coverNFT.connect(members[0]).mint(members[1].address, 0);
-    await coverNFT.connect(members[1]).setApprovalForAll(members[0].address, true);
-    expect(await coverNFT.isApprovedOrOwner(members[0].address, 0)).to.be.equal(true);
-    expect(await coverNFT.isApprovedOrOwner(members[1].address, 0)).to.be.equal(true);
+    const {
+      members: [operator, nftOwner],
+    } = this.accounts;
+    await coverNFT.setMockOperator(operator.address);
+    await coverNFT.connect(operator).mint(nftOwner.address, 0);
+    await coverNFT.connect(nftOwner).setApprovalForAll(operator.address, true);
```

Comment:
let's use a different address here and not operator.

arbitrary address so there's no confusion that the operator has special permissions.